lms/djangoapps/course_api/blocks/permissions.py

```diff
@@ -28,7 +28,14 @@ def can_access_self_blocks(requesting_user, course_key):
     """
     Returns whether the requesting_user can access own blocks.
     """
-    return (
+    user_is_enrolled_or_staff = (  # pylint: disable=consider-using-ternary
         (requesting_user.id and CourseEnrollment.is_enrolled(requesting_user, course_key)) or
         has_access(requesting_user, CourseStaffRole.ROLE, course_key)
     )
+    if user_is_enrolled_or_staff:
+        return user_is_enrolled_or_staff
+    else:
```

Comment:
Similar to the earlier comment, this one will be even easier to read w/out the `else` (keyword), since it has further indent levels.